src/ulohy/podminky_a_cykly.md

```diff
@@ -26,9 +26,9 @@ dan = 0
 ```
 
 ```
-utraceno = 10412
+utraceno = 104120
 mzda = 41265
-dan = 1561
+dan = 15618
 ```
 
 ```
```

Comment:
The second rule is already here :) We should reorder it and apply the third rule with e.g. `utraceno = 150000` and `dan = 22500`.